ddp-workspace/projects/ddp-dsm-ui/src/app/sequencing-order/sequencing-order.component.html

```diff
@@ -79,7 +79,7 @@
     </div>
     <div class="app-modal-footer">
       <button (click)="closeModal()" class="btn btn-default" type="button">Close</button>
-      <button (click)="submitOrders()" *ngIf="!this.checkError()" class="btn btn-default" type="button">Submit</button>
+      <button (click)="submitOrders()" *ngIf="!this.checkError()" class="btn btn-default" type="button" [disabled]="canNotSubmitOrder()">Submit</button>
```

Comment:
There is a guard in the backend that doesn't let the orders go through without proper "submit orders" access role, but I am also removing the buttons from the view if they  only have the view role

Reply:
Just so I understand, this is a 'drive-by' fix... not required for this task, but something you noticed that was incorrect?

Reply:
It is required in this task that the user with only this new permission, should not be able to submit an order. There is a backend check for this, but I also removed the submit button